packages/fiori/test/pages/Wizard.html

```diff
@@ -673,6 +673,80 @@ <h2>Wizard non-standard 3</h2>
 		</ui5-wizard>
 	</ui5-dialog>
 
+	<h1>FIX EXAMPLE</h1>
```

Comment:
Avoid usage of h1 to label a sample section. h1 by intend should be used as a heading of a whole page. Do not use "FIX EXAMPLE" but a more descriptive name for the sample below.